docs/CHANGELOG.asciidoc

```diff
@@ -10,10 +10,496 @@
+
+*Impact* +
+Before you upgrade, remove the settings from kibana.yml, then configure the settings in <<logs-app,Logs>>.
+====
+
+[discrete]
+[[breaking-61302]]
+.Removes configurable fields in settings
+[%collapsible]
+====
+*Details* +
+The *Logs* and *Metrics* configurable fields settings have been removed. For more information, refer to {kibana-pull}61302[#61302].
+
+*Impact* +
+Configure the settings in https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[ECS].
+====
+
+[float]
+[[machine-learning-breaking-changes-8.0.0-beta1]]
+==== Machine learning
+      
+[discrete]
+[[breaking-115444]]
+.Grants access to machine learning features when base privileges are used
+[%collapsible]
+====
+*Details* +
+Machine learning features are included as base privileges. For more information, refer to {kibana-pull}115444[#115444].
+
+*Impact* +
+If you do not want to grant users privileges to machine learning features, update <<xpack-security,*Users* and *Roles*>>.
+====
+
+[float]
+[[maps-breaking-changes-8.0.0-beta1]]
+==== Maps
+
+[discrete]
+[[breaking-116184]]
+.Removes proxyElasticMapsServiceInMaps
+[%collapsible]
+====
+*Details* +
+The `map.proxyElasticMapsServiceInMaps` setting has been removed. For more information, refer to {kibana-pull}116184[#116184].
+
+*Impact* +
+Install the on-prem version of the <<maps-connect-to-ems,Elastic Maps Service>>, which is a Docker service that resides in the Elastic Docker registry, in an accessible location on your internal network. When you complete the installation, update kibana.yml to point to the service.
+====
+
+[float]
+[[operations-breaking-changes-8.0.0-beta1]]
+==== Operations 
+
+[discrete]
+[[breaking-111535]]
+.Removes environment variables
+[%collapsible]
+====
+*Details* +
+The `CONFIG_PATH` and `DATA_PATH` environment variables have been removed. For more information, refer to {kibana-pull}111535[#111535].
+
+*Impact* +
+Replace the `CONFIG_PATH` environment variable with `KBN_PATH_CONF`, and replace `DATA_PATH` with the `path.data` setting.
+====
+
+[float]
+[[platform-breaking-changes-8.0.0-beta1]]
+==== Platform 
+      
+[discrete]
+[[breaking-114379]]
+.Removes supports for csp.rules configuration
+[%collapsible]
+====
+*Details* +
+Support for the `csp.rules` configuration property has been removed. For more information, refer to {kibana-pull}114379[#114379].
+
+*Impact* +
+Configuring the default `csp.script_src`, `csp.workers_src`, and `csp.style_src` values is not required.
+====
+
+[discrete]
+[[breaking-113653]]
+.Removes unused and renamed deprecated core settings and deprecated settings from core plugins
+[%collapsible]
+====
+*Details* +
+The deprecation notice for `server.cors` has changed from `level:critical` to `level:warning`. 
+
+Support for the following configuration settings has been removed:
+
+* `newsfeed.defaultLanguage`
+* `xpack.banners.placement`
```

Comment:
Unlike other items in this list, support for this specific config has not actually been removed, just one of the values it accepts has changed (as explained in the details below). 

We might want to relocate this item elsewhere to prevent confusion.